auth/auth.go

```diff
@@ -98,11 +98,19 @@ func NewClient(ctx context.Context, conf *internal.AuthConfig) (*Client, error)
 	if err != nil {
 		return nil, err
 	}
+
+	version := "Go/Admin/" + conf.Version
 	return &Client{
+		userManagementClient: userManagementClient{
+			baseURL:    idToolkitEndpoint,
+			projectID:  conf.ProjectID,
+			version:    version,
```

Comment:
It feels like a code smell to duplicate `version` in two places, although it might not be a bad idea. It's unclear without knowing more about userManagementClient.

Reply:
In the future we will have `userManagementClient` implement all the user management APIs. In that world we won't need `auth.Client.version` at all. Will add a comment to make it clear that this is just an interim step.